File: DecoAroms-Fronted/src/utils/medioPago.js

export const MEDIOS_PAGO_MAP = {
    MERCADO_PAGO: { value: 'MERCADO_PAGO', label: 'Mercado Pago' },
    BCI: { value: 'BCI', label: 'Bci' },
    TRANSFERENCIA: { value: 'TRANSFERENCIA', label: 'Transferencia' },
    EFECTIVO: { value: 'EFECTIVO', label: 'Efectivo' },
    BOTON_DE_PAGO: { value: 'BOTON_DE_PAGO', label: 'Botón de Pago' },
    POST: { value: 'POST', label: 'Post' },
};

// Transformamos el objeto en un array para usarlo fácilmente en .map() en los componentes
export const MEDIOS_PAGO = Object.values(MEDIOS_PAGO_MAP);

File: DecoAromasPOS/src/main/java/com/decoaromas/decoaromaspos/enums/MedioPago.java
package com.decoaromas.decoaromaspos.enums;

import lombok.Getter;

@Getter
public enum MedioPago {
    MERCADO_PAGO("Mercado Pago"),
    BCI("Bci"),
    TRANSFERENCIA("Transferencia"),
    EFECTIVO("Efectivo"),
    BOTON_DE_PAGO("Botón De Pago"),
    POST("Post");

    private final String nombreParaUi;

    MedioPago(String nombreParaUi) {
        this.nombreParaUi = nombreParaUi;
    }
}
